docs/source/supported-libraries/index.rst

```diff
@@ -0,0 +1,5 @@
+Supported libraries
+===================
+
+.. toctree::
+   django-filter
```

Comment:
Does this mean libraries that support running the test or libraries that work with django-mongodb-backend?

Reply:
The latter. Running the tests with the CLI is a convenience for us, but defining what support means in general is the goal.